Kitodo/src/main/java/org/kitodo/production/services/data/PropertyService.java

```diff
@@ -171,17 +130,8 @@ private QueryBuilder getQueryForType(String type) {
      *            of the searched property
      * @return list of JSON objects with properties
      */
-    List<Map<String,Object>> findByValue(String value, String type, boolean contains) throws DataException {
-        return findProperty(PropertyTypeField.VALUE.getKey(), value, type, contains);
-    }
-
-    private List<Map<String,Object>> findProperty(String key, String value, String type, boolean contains) throws DataException {
-        BoolQueryBuilder query = new BoolQueryBuilder();
-        query.must(createSimpleQuery(key, value, contains, Operator.AND));
-        if (Objects.nonNull(type)) {
-            query.must(createSimpleQuery(PropertyTypeField.TYPE.getKey(), type, true));
-        }
-        return findDocuments(query);
+    List<Property> findByValue(String value, String type, boolean contains) {
+        return getByQuery("from Property as property where property.value='" + value + "'");
```

Comment:
Please use `getByQuery` with second parameter with list of to map values. If not this is introducing a possible SQL injection.